xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp

```diff
@@ -974,8 +974,7 @@ void CGUIDialogPVRTimerSettings::ChannelsFiller(const SettingConstPtr& setting,
             !pThis->m_timerType->SupportsAnyChannel())
           continue;
 
-        list.emplace_back(
-            IntegerSettingOption(channelEntry.second.description, channelEntry.first));
+        list.emplace_back(channelEntry.second.description, channelEntry.first);
```

Comment:
Please revert this, it is not given that `IntegerSettingOption` will actually stay a `std::pair` forever.